.openpublishing.redirection.json

```diff
@@ -2859,6 +2859,14 @@
             "source_path": "docs/fundamentals/code-analysis/style-rules/ide0009.md",
             "redirect_url": "/dotnet/fundamentals/code-analysis/style-rules/ide0003-ide0009.md"
         },
+        {
+            "source_path": "docs/fundamentals/code-analysis/style-rules/ide0047.md",
```

Comment:
You don't need to add the redirects since ide0047.md and ide0048.md were never published.

Reply:
They will be used by the analyzers themselves. We want each analyzer to use automatic assigned help links based on its rule ID.